### PR TITLE
Add Healthcare

### PR DESCRIPTION
**Pull request by MatthewVita**
https://github.com/kakoni/awesome-health

A curated list of awesome open health software, libraries, tools and resources. This guide is very nice for working professionals and healthcare providers looking for open source health solutions. I hope you consider adding it!

# By submitting this pull request I confirm I've read and complied with the below requirements.

Failure to properly do so will just result in the pull request being closed and everyone's time wasted. Please read it twice. Most people miss many things.

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title. For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should include a short description about the list project/theme (not the list itself), and should be added at the bottom of the appropriate category.
- The list I added
	- **has been around for at least 30 days,**
	- is the result of hard work and is the best I could possibly produce,
	- is a non-generated Markdown file in a GitHub repo,
	- is not a duplicate,
	- includes a succinct description of the project/theme at the top ([example](https://github.com/willempienaar/awesome-quantified-self)),
	- includes a project logo/illustration (whenever possible) at the top right of the readme ([example](https://github.com/sindresorhus/awesome-electron)). (The logo should link to the project website or any relevant website. The logo should be high-DPI, so set it to maximum half the width of the original image.),
	- only has awesome items **with descriptions** *(Awesome lists are curations of the best, not of everything)*,
	- contains the [awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
	- has a Table of Contents section **named `Contents`** as the first section,
	- has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license) *(Meaning something like CC0, NOT a code licence like MIT, BSD, Apache, etc)* *(If you use a license badge, it should be SVG, not PNG)*,
	- has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines),
	- doesn't include a Travis badge,
	- has been checked for inconsistent formatting, spelling, and grammar:
		- Each link description starts with an uppercase character and ends with a period.
		- Drop all the `A` / `An` prefixes in the descriptions.
		- Consistent naming. For example, `Node.js`, not `NodeJS` or `node.js`.

### CODE REVIEW

**Comment by MatthewVita** [comment]
@sindresorhus Just pinging you to see if this has been reviewed. For whatever it's worth, I do think this a pretty harmless addition.

Thanks for all you do with this project!

**Comment by legodude17** [comment]
Not followed:
 * Each link description starts with an uppercase character and ends with a period.

Also, add the topics `awesome-list`, `awesome`, and `list`.

**Comment by MatthewVita** [comment]
Hi @legodude17,

Thanks for the review.

- I just committed (https://github.com/kakoni/awesome-health/commit/8c4a78234cf32461dd653019c04e1447ae01f847) the formatting fixes.
- I have asked the project admin, Karri, to add those topics via https://github.com/kakoni/awesome-health/issues/21

I am sure that Karri will add those topics in a timely fashion. Feel free to bring in this PR.

Thanks,
Matthew

**Comment by legodude17** [comment]
@MatthewVita I don't have permission to merge PR's, I am just helping out @sindresorhus.

@sindresorhus Anything else for this PR?

**Comment by MatthewVita** [comment]
Sounds like you should be made a contributor then 👍

**Comment by jcfr** [comment]
@legodude17 and @sindresorhus Thanks for organizing the awesome lists 👍 

Having just contributed to the [awesome-health](https://github.com/kakoni/awesome-health) list, my contribution was well received by @MatthewVita  and we had a productive exchange.

Let me know what I can do to help integrate the health list to the top-level list.

Have a great weekend,
Jc

**Comment by MatthewVita** [comment]
@sindresorhus,

Just rebased the branch.

As @jcfr said, let us know what can be done to help integrate this. I _would_ offer to help maintain this repo, but I am maintaining 2 other projects and wouldn't be able to find the time. However, it seems like @legodude17 has proven that he has what it takes to be a maintainer.

Thanks,
Matthew

**Comment by sindresorhus** [comment]
`Awesome Health` should not be in the TOC.

`OS X` => `macOS`

`A curated list of medical data` => `Curated list of medical data`

---

Is `Health` really the best title for this list? It seems more like a `Medical` list or something. Health is a lot more generic than this list. It could be personal health and how to be more healthy.

---

What's the goal of the list? That should really be stated at the top of the readme. And who are the target group? Kinda seems like the list is for people starting their own hospital.

**Comment by MatthewVita** [comment]
Hi @sindresorhus,

Thanks for the response. I have adjusted the heading to:

> Curated list of awesome open source healthcare software, libraries, tools and resources. Each link has been vetted to ensure the project is active and provides value to healthcare facilities, providers, developers, policy experts, and/or research scientists.

Have also removed the redundant root note from the ToC.

I can see how `awesome-health` is a very vague title. I believe `awesome-healthcare` is a more specific title. I will talk to the other maintainer to see if this can be changed. Thank you in advance for your patience.

Thanks,
Matthew

**Comment by MatthewVita** [comment]
Hi @sindresorhus,

I am happy to inform you that this PR is ready to go!

Thanks,
Matthew

File: readme.md [hunk 1]
```diff
@@ -515,6 +515,7 @@
 - [Framer](https://github.com/podo/awesome-framer) - Prototyping interactive UI designs.
 - [Markdown](https://github.com/BubuAnabelas/awesome-markdown) - Markup language.
 - [Dev Fun](https://github.com/mislavcimpersak/awesome-dev-fun) - Funny developer projects.
+- [Healthcare](https://github.com/kakoni/awesome-healthcare) - Open source healthcare software, libraries, tools and resources.
```
Comment (sindresorhus) [inline readme.md]: > The entry in the Awesome list should include a short description about the list project/theme **(not the list itself)**

**Comment by MatthewVita** [comment]
Hi @sindresorhus,

I see what you mean and have made the appropriate changes.

Your eye on quality is appreciated.

Thanks,
Matthew

**Comment by MatthewVita** [comment]
Hi @sindresorhus,

Was wondering if this is ready to go? Please let me know if you'd like anything tweaked.

Thanks,
Matthew

**Comment by MatthewVita** [comment]
Thanks!

**Comment by jcfr** [comment]
Thanks @sindresorhus and @MatthewVita

**Pull request merged**